Loenn/lang/en_gb.lang

```diff
@@ -332,6 +332,11 @@ entities.CommunalHelper/ConnectedMoveBlock.attributes.description.pressedColor=T
 entities.CommunalHelper/ConnectedMoveBlock.attributes.description.breakColor=The color of the arrow when breaking.
 entities.CommunalHelper/ConnectedMoveBlock.attributes.description.outline=Gives this Move Block an outline.
 
+# Connected Temple Cracked Block
+entities.CommunalHelper/ConnectedTempleCrackedBlock.placements.name.normal=Connected Temple Cracked Block
+entities.CommunalHelper/ConnectedTempleCrackedBlock.placements.name.persistent=Connected Temple Cracked Block (Persistent)
+entities.CommunalHelper/ConnectedTempleCrackedBlock.attributes.description.persistent=Persistent blocks will stay broken after dying or leaving the room.
```

Comment:
You forgot to add this tooltip to the Ahorn `en_gb.lang` file.

Reply:
What tooltip? The only attribute besides position/size is the persistent checkbox and that's already in there

Reply:
OH WAIT you said Ahorn not Loenn. I'll go do that...

Reply:
I can't find it. The PR doesn't list any change in `Ahorn/lang/en_gb.lang` either.

Reply:
ok done